FRAuth/FRAuth/Callbacks/FRAppIntegrityCallback.swift

```diff
@@ -117,32 +133,33 @@ public class FRAppIntegrityCallback: MultipleValuesCallback {
     /// Sets `keyId` value in callback response
     /// - Parameter keyId: Base64 String value of keyId
     public func setkeyId(_ keyId: String) {
-        self.inputValues[self.keyId] = keyId
+        self.inputValues[self.keyIdKey] = keyId
     }
     
     /// Sets `token` value in callback response
     /// - Parameter token: Base64 String value of verification
-    public func setVerification(_ token: String) {
-        self.inputValues[self.assertToken] = token
+    public func setAssertion(_ token: String) {
+        self.inputValues[self.tokenKey] = token
     }
     
     /// Sets `clientData` value in callback response
     /// - Parameter clientData: Base64 String value of clientData
     public func setClientData(_ clientData: String) {
-        self.inputValues[self.clientData] = clientData
+        self.inputValues[self.clientDataKey] = clientData
     }
     
     /// Attest the device for iOS14 and above devices
     /// - Throws: `FRDeviceCheckAPIFailure`
     /// - Parameter attestation: Optional Protocol for providing a ``FRAppAttestation`` to implement own attestation
     @available(iOS 14.0, *)
-    public func attest() async throws {
+    open func requestIntegrityToken() async throws {
         do {
-            let result = try await FRAppAttestDomainModal.shared.attest(challenge: challenge)
-            self.setAttestation(result.attestKey)
-            self.setVerification(result.assertKey)
+            let result = try await FRAppAttestDomainModal.shared.requestIntegrityToken(challenge: challenge)
+            self.setAttestation(result.appAttestKey)
+            self.setAssertion(result.assertKey ?? "")
```

Comment:
Should we through if you have no key? Or is there a valid reason that would happen?

Reply:
yes, that value can be either empty or have a value ..AM needs that information 

but i can modify that logic like this. good find

if let assertkey = result.assertKey {
 self.setAssertion(assertkey)
}